devenv/sync.py

```diff
@@ -69,6 +69,9 @@ def main(context: dict[str, str]) -> int:
     repo = context["repo"]
     reporoot = context["reporoot"]
 
+    # don't want this to be accidentally run from getsentry
+    assert repo == "sentry"
```

Comment:
I would do `asssert repo == "sentry", repo` just so it's clearer in the error message